addons/nuxeo-retention/nuxeo-retention-web-ui/src/main/resources/web/nuxeo.war/ui/document/retentionrule/nuxeo-retentionrule-create-layout.html

```diff
@@ -140,26 +150,24 @@
     </nuxeo-card>
 
     <nuxeo-card heading="[[i18n('retention.rule.label.duration.heading')]]">
+      <div class="retentionDuration">[[i18n('retention.rule.label.duration.subHeading')]]</div>
       <div class="layout horizontal justified flex">
         <nuxeo-input role="widget" name="years"
           value="{{document.properties.retention_def:durationYears}}"
           label="[[i18n('retention.rule.label.duration.years')]]"
           min="0"
-          required
```

Comment:
How does this play with form validation? By adding the CSS rules you are basically just doing the style part of the required, but since the field is no longer required, doesn't it compromise form validation? Does it allow you to create a retention rule without any of the retention duration fields?

Reply:
the validation is being guaranteed by the validate method - so if you keep all fields empty it will fail the validation and not allow you to create the rule.

Reply:
But you no longer have the `*` to display the field as required. If that is not required, then all good for me 😄